packages/Autocomplete/src/Autocomplete.tsx

```diff
@@ -20,13 +21,15 @@ const Autocomplete: React.SFC<IAutocompleteProps> = ({
   label,
   placeholder,
   selected,
+  onChange,
   inputProps,
 }) => {
   const [inputValue, setInputValue] = useState<string>('');
   const [showSelect, setShowSelect] = useState<boolean>(false);
 
   const handleChange = ({ value }: { value: string }): void => {
     setShowSelect(true);
+    onChange && onChange(value);
```

Comment:
We will want to debounce the changes so we don't send too many requests to the API e.g. it will send the request after 500ms not every key press will this allow us to do that? Can we control that outside of the component?

Reply:
yes It is better to control the bounce in Blaze outside of this component